src/main/java/org/threadly/concurrent/PriorityScheduler.java

```diff
@@ -745,7 +722,12 @@ public int getCurrentPoolSize() {
      */
     public int getCurrentRunningCount() {
       synchronized (workersLock) {
-        return currentPoolSize - availableWorkers.size();
+        int workerOutQty = currentPoolSize - availableWorkers.size();
```

Comment:
This math changes because if there is a worker out (ie no thread blocked waiting for a worker), then it will be waiting in the consumer, not running a task.